src/redux.js

```diff
@@ -1228,8 +1235,13 @@ export function getPanelButtons(state) {
       : { panel: "continue", text: "Continue" };
   } else if (state.currentPanel === "saveModel") {
     prev = { panel: "results", text: "Back" };
-    next = isPanelAvailable(state, "save")
-      ? { panel: "save", text: "Finish" }
+    next = isPanelAvailable(state, "modelSummary")
+      ? { panel: "modelSummary", text: "Save" }
+      : null;
+  } else if (state.currentPanel === "modelSummary") {
+    prev = { panel: "saveModel", text: "Back" };
+    next = isPanelAvailable(state, "finish")
+      ? { panel: "finish", text: "Finish" }
       : null;
```

Comment:
What's the user experience if the save fails?

Reply:
Good call! I changed this functionality. If the save fails, they'll see a failure message and can click Save to try again. If it succeeds, the Save button becomes Continue.

Reply:
These buttons are designed as navigation buttons rather than action buttons with no navigation.  I think that once Save is pressed, a successful save should navigate to the next screen rather than turn into a different button.